pkg/products/monitoringspec/reconciler.go

```diff
@@ -39,6 +40,79 @@ const (
 	clonedServiceMonitorLabelValue            = "true"
 )
 
+//Remove this section, it's used for a temporary function to clean up obsolete resources
+//TODO remove from here ---------------------------------------------------
+type obsoleteObjectStruct struct {
+	name       string
+	namespace  string
+	objectType string
+}
+
+var obsoleteObjects = []obsoleteObjectStruct{
+	{
+		name:       "syndesis-infra-api-dashboard",
+		namespace:  "redhat-rhmi-middleware-monitoring",
```

Comment:
Not entirely sure if this is the correct namespace/ will be able to test this once we can spin up clusters again.